libs/ledgerjs/packages/hw-app-exchange/src/Exchange.ts

```diff
@@ -59,6 +60,19 @@ const curves = {
   secp256r1: 0x01,
 };
 export type PayloadSignatureComputedFormat = "raw" | "jws";
+export type PayloadCore = {
```

Comment:
This type should be define in the same file as the `decodePayloadProtobuf` function (in a different file).
The name should be more precise with "swap" terms